src/zope/app/wsgi/testlayer.py

```diff
@@ -14,17 +14,20 @@
 import base64
 import http.client as httpclient
 import re
+import typing
 import xmlrpc.client
 from io import BytesIO
 
 import transaction
+import webtest
```

Comment:
Should we declare the `webtest` dependency in `setup.py`'s `install_requires`?

We currently have the `webtest` dependency in `tests_require`. But without it in `instal_requires` packages using the new function would need to depend on `zope.app.wsgi[test]` (instead of simple `zope.app.wsgi`) in their `tests_require`.

Reply:
I see that depending on `zope.app.wsgi[testlayer]` (instead of `zope.app.wsgi[test]`) would be enough for other packages to use `testlayer` for their tests.

Reply:
I think that was the idea: to be able to use the testlayer functionality, depend on that extra instead of the `test` one.